models/games.model.js

```diff
@@ -7,18 +7,27 @@ exports.selectAllCategories = () => {
 };
 
 exports.selectReviewById = (reviewId) => {
-  return db
-    .query("SELECT * FROM reviews WHERE  review_id = $1", [reviewId])
-    .then((review) => {
-      if (review.rows.length === 0) {
-        return Promise.reject({
-          status: 404,
-          msg: "review does not exist",
-        });
-      } else {
-        return review.rows[0];
-      }
-    });
+  return (
+    db
+      //.query(`SELECT * FROM reviews WHERE  review_id = $1`, [reviewId])
+      .query(
+        `SELECT reviews.*,
```

Comment:
I dont think these changes are relevant to deleting a comment, please be carful not to add changes to a branch that are not relevant to it